erikbenedict/hype-hub
# Feature/delete routes

## Delete buttons

- Added delete buttons on the topics and achievement cards
- created routes for each
- added js logic to handle delete functionality

## TEST

1. add a topic on the dashboard, and then click on the 'x' to delete the topic card
2. see that a confirmation pops up to confirm the delete
3. add an achievement under a topic
4. repeat same steps as in '1' and '2' for achievement card
